### ISSUE TITLE
The comparison of sbyte array is wrong

### ISSUE THREAD

**Issue by taks** [GregFinzer/Compare-Net-Objects#202]
The following sbyte array comparisons return wrong results.
```
var compareLogic = new KellermanSoftware.CompareNetObjects.CompareLogic();
var actual = new sbyte[] { 2, 1 };
var expected = new sbyte[] { 1, 2 };
var result = compareLogic.Compare(expected, actual);
```
`result.AreEqual` is true, although false is expected.

**Comment by GregFinzer** [comment]
@taks The Compare .NET Objects library does not support sbyte.  If you would like to implement this new feature then please create it with tests and create a pull request.

**Comment by taks** [comment]
Ok, I've created a PR.

**Comment by GregFinzer** [comment]
Merged thanks so much

**Comment by GregFinzer** [comment]
Released in Version 4.67 on NuGet
